sunpy/net/vso/vso.py

```diff
@@ -793,7 +793,7 @@ def download_all(self, response, methods, downloader, path, qr, info=None):
                             qr[dataitem.fileiditem.fileid[0]]
                         )
                     except NoData:
-                        results.add_error('', DownloadFailed(dresponse))
+                        results.add_error('', '', DownloadFailed(dresponse))
```

Comment:
What are all these here for?

Reply:
There was an API change in parfive that broke this code.